core/finality-grandpa/src/lib.rs

```diff
@@ -794,6 +576,10 @@ impl<B, E, Block: BlockT<Hash=H256>, N, RA> voter::Environment<Block::Hash, Numb
 			} else {
 				info!("Applying GRANDPA set change to new set {:?}", set_ref);
 			}
+
+			// TODO [now]: write commit to DB in a way where it can be fetched for syncing
```

Comment:
No committing to `//TODO` to master unless there is a link to an issue. Do we have an issue for that yet? Please link.

Reply:
https://github.com/paritytech/substrate/issues/1026 would be the one. @andresilva is working on that, but mostly within the scope of a round-skipping protocol. I've removed the TODO and just ignore the `commit` parameter within this PR now.